InvasivesBC/ViewControllers/BaseViewController.swift

```diff
@@ -0,0 +1,237 @@
+//
+//  BaseViewController.swift
+//  ipad
+//
+//  Created by Amir Shayegh on 2019-10-29.
+//  Copyright © 2019 Amir Shayegh. All rights reserved.
+//
+
+import UIKit
+import DatePicker
+
+protocol InputDelegate: class {
```

Comment:
I added the same protocol 'FieldAuxViewPresenterDelegate'. Path InvasivesBC/FormFramework/Core/BaseCell/BaseFieldCell.swift